holidays/locale/be/LC_MESSAGES/BY.po

```diff
@@ -12,8 +12,7 @@ msgstr ""
 "MIME-Version: 1.0\n"
 "Content-Type: text/plain; charset=UTF-8\n"
 "Content-Transfer-Encoding: 8bit\n"
-"Plural-Forms: nplurals=3; plural=(n%10==1 && n%100!=11 ? 0 : n%10>=2 && n%10<=4 && (n%100<12 || "
```

Comment:
It's redundant for this PR (and doesn't match max line length = 99)

Reply:
Oof, that came from `make l10n` and must have been accidentally included in the commit by accident, removing it right now